README.md

```diff
@@ -1 +1,185 @@
-# rails_blog
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] Kindly remove the template logo and do same for all your future projects.  You can use your personal logo if you have one